exceptions/guild.py

```diff
@@ -0,0 +1,44 @@
+"""Custom exceptions for guild related cogs."""
+
+from collections.abc import Sequence
+
+__all__: Sequence[str] = (
+    "DiscordMemberNotInMainGuildError",
+    "EveryoneRoleCouldNotBeRetrievedError",
+)
+
+from classproperties import classproperty
+
+from exceptions import BaseErrorWithErrorCode, BaseTeXBotError
```

Comment:
Use relative imports for importing base exception classes. I.e. `from .base import ...` rather than ~`from exceptions import ...`~ or ~`from exceptions.base import ...`~